src/templates/list-table-block/index.vue

```diff
@@ -576,7 +576,7 @@
     loadData({ page: 1, ...query })
   }
 
-  defineExpose({ reload, handleExportTableDataToCSV })
+  defineExpose({ reload, handleExportTableDataToCSV, data })
```

Comment:
Why are we doing this exactly?

Reply:
Eu preciso ter acesso dos dados da tabela a partir da ref dela, pq o azion sso não existe na api, dessa forma quando eu quero setar o azion sso (opcao default q nao existe na api) como padrao, eu preciso percorrer toda a tabela pra achar qual está ativo no momento e ai sim envia-lo para o serviço para defini-lo como inativo 🤡